### PR TITLE
Fix ajax refresh for listing pages

### PR DESCRIPTION
**Pull request by julienbourdeau**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Some data weren't updated properly + markup was in core.js instead of theme.js |
| Type? | bug fix |
| Category? | FO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1644 |
| How to test? | Use the faceted search in the sidebar and see if it works |

### CODE REVIEW

**Comment by aleeks** [comment]
@julienbourdeau can you rebase please, i'm sorry!

**Comment by maximebiloe** [comment]
Don't forget to report it on `StarterTheme` please

File: themes/classic/templates/catalog/_partials/products.tpl [hunk 1]
```diff
@@ -1,55 +1,20 @@
...
-        <i class="material-icons">&#xE316;</i>
-      </a>
-    </div>
-  {else}
-    {include file='errors/not-found.tpl'}
-  {/if}
-</section>
+  <div class="hidden-md-up text-xs-right up">
+    <a href="#header" class="btn btn-secondary">
+      Back to top
```
Comment (aleeks) [inline themes/classic/templates/catalog/_partials/products.tpl]: @julienbourdeau  Where is my translation! :smiling_imp:

**Review by aleeks — CHANGES_REQUESTED** [review]

File: themes/classic/templates/catalog/_partials/products.tpl (hunk 1 above)
Comment (mickaelandrieu) [inline themes/classic/templates/catalog/_partials/products.tpl]: Shame on you 😈 !

**Review by aleeks — APPROVED** [review]

**Comment by aleeks** [comment]
thank you @julienbourdeau  !

**Pull request merged**